scripts/train_summary.js

```diff
@@ -25,9 +25,10 @@ var TRAIN_REPOS = {
   'fxa-customs-server': {},
   'fxa-profile-server': {},
   'fxa-basket-proxy': {},
+  'fxa-email-service': {},
 }
 
-console.log("Updating repos...")
+console.warn("Updating repos...")
```

Comment:
I changed these from logs to warns so that they would not end up in the output file when called like:

```bash
node ./scripts/train_summary.js 116 > train-116.md
```

Reply:
Could we add docs somewhere on how to run the script?  we have some here https://github.com/mozilla/fxa#admin-operations